### DISCUSSION TITLE
smtp supports ssl

### DISCUSSION THREAD

**Discussion by qingfengfenga** [juanluisbaptiste/docker-postfix#72] [Q&A]
Sorry, I want to forward the mail received by one mail account to another smtp server, but it doesn't seem to support SSL, and the source server is forced to use SSL. Is it because I have a misunderstanding about the function of this project.

**Comment by juanluisbaptiste** [comment]
Hi @qingfengfenga I think you are misunderstanding the usage of this container. Its purpose is to allow other containers send emails using another email server as its smtp relay.
